chart/orkestra/crds/orkestra.azure.microsoft.com_applicationgroups.yaml

```diff
@@ -58,6 +58,12 @@ spec:
                     name:
                       description: Name of the application
                       type: string
+                    order:
```

Comment:
There is a difference between these files, can you make sure they are the same spec?

Reply:
will do!